public/media-hunt/mediahunt.js

```diff
@@ -0,0 +1,19 @@
+const { instance } = require("./client.js");
+
+//get popular movies
+
+async function getData(path) {
+  try {
+    const { data } = await instance.get(path);
+    return data;
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function run() {
+  const allMedia = await getData("movie/popular?language=en-US&page=1");
+  console.log(allMedia);
+}
+
+module.exports = { run };
```

Comment:
This entire file is incorrect. 
Please create separate files for movies and tv and conf., which will have their own api fetch methods.
Also, group them maybe in a folder called "http", and have your axios instance also generated there.

Example:
```
// movie.js

// Import instance and required path for movies from your ali scheme 

const getMovies = async () => {
  const { data } = await instance.get(movieUrl);
  return data;
}

...

module.exports = { getMovies, ... }
```

But also let's use "discover" API calls to get movies and tv series.